dashboard/test/ui/features/star_labs/custom_blocks.feature

```diff
@@ -0,0 +1,13 @@
+@chrome
+Feature: Google Blockly Custom Blocks
+
+Background:
+  Given I create a levelbuilder named "Simone"
+
```

Comment:
Is it possible to create a map of labs and whether they are on google blockly or CDO blocly? Just curious.

Reply:
mmm yeah you can tell which blockly a lab uses based on this `uses_google_blockly?` method in the level type:

https://github.com/code-dot-org/code-dot-org/blob/63f721c2826ce7dde1614aabbb84986900cf4afc/dashboard/app/models/levels/dancelab.rb#L57-L59

That said, these tests are for block "pools" rather than a specific lab, and pools can be used by multiple labs that are on different blockly versions (eg, I believe the GamelabJr (aka spritelab) pool is used by both Poetry and Sprite Lab)).

If would be slick if we could easily associate a "pool" with the labs that use them, and then run this test for each pool associated with any lab that uses google blockly. Feels like a bit of overkill at this point though -- unfortunately, it doesn't look like we have a Pool model (just Block) that would make this association easy, not sure why.